docs/proposals/prop-2-releases.md

```diff
@@ -0,0 +1,72 @@
+Proposal: 1
+Title: Releasing Anonlink
+Author: Brian Thorne
+Status: Draft
+Created: 2019-11-5
+
+## Abstract
+
+This proposal outlines a design for automated packaging and releasing of anonlink using our primary CI/CD system - 
+Microsoft's Azure DevOps [^1].
+
+## Motivation 
+
+An automated process for releasing Anonlink allows for more frequent releases and ensures that releases are
+packaged, reviewed and communicated consistently.
+
```

Comment:
It may be good to state that the CI is already building all the required packages automatically.